pandas/core/internals/managers.py

```diff
@@ -0,0 +1,1892 @@
+# -*- coding: utf-8 -*-
+from functools import partial
+import itertools
+import operator
+
+import numpy as np
+
+from pandas._libs import lib, internals as libinternals
+
+from pandas.util._validators import validate_bool_kwarg
+
+from pandas.compat import range, map, zip
+
+import pandas.core.dtypes.generic as gt
```

Comment:
I noticed this usage in `tests.dtypes.test_generic`, think it's a nice way to cut down massive import blocks/namespaces.